pulp_rpm/app/models/package.py

```diff
@@ -288,8 +288,7 @@ def createrepo_to_dict(cls, package):
             PULP_PACKAGE_ATTRS.ENHANCES: getattr(package, CR_PACKAGE_ATTRS.ENHANCES, []),
             PULP_PACKAGE_ATTRS.EPOCH: getattr(package, CR_PACKAGE_ATTRS.EPOCH) or "",
             PULP_PACKAGE_ATTRS.FILES: getattr(package, CR_PACKAGE_ATTRS.FILES, []),
-            PULP_PACKAGE_ATTRS.LOCATION_BASE: getattr(package, CR_PACKAGE_ATTRS.LOCATION_BASE)
-            or "",
+            PULP_PACKAGE_ATTRS.LOCATION_BASE: "",  # TODO, delete this entirely
```

Comment:
We cannot really do anything useful with "location_base", we should delete it entirely.  Including from the models and  REST API if possible.  But that can be deferred for now.

Reply:
Do I understand correctly, that after this change, old records which had location base will have some data in the database and new ones won't after this change?
It's not important, just inconsistent. Maybe it's worth taking care when we remove location base entirely.

Reply:
Correct.  I agree, we should probably do it soon-ish.  At least the model removal - we should probably move slower for the REST API.